pkgs/os-specific/linux/bluez/default.nix

```diff
@@ -1,13 +1,13 @@
 { stdenv, fetchurl, pkgconfig, dbus, glib, alsaLib,
-  python3, readline, udev, libical, systemd, fetchpatch,
+  python3, readline, udev, libical, systemd,
   enableWiimote ? false, enableMidi ? false, enableSixaxis ? false }:
```

Comment:
Newer bluez versions have support for additional optional features, at least

```
--enable-nfc
--enable-mesh
--enable-sap
````

It might be out of the scope of this PR though.